keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlAgentLockRepository.kt

```diff
@@ -87,4 +38,12 @@ class SqlAgentLockRepository(
 
     return changed == 1
   }
+
+  override fun getLockedAgents(): List<String> {
+    return sqlRetry.withRetry(RetryCategory.READ) {
```

Comment:
nit: will you add an import for `RetryCategory` so this becomes `return sqlRetry.withRetry(READ) {`?

Reply:
Done!